bin/ansible_tower-operations

```diff
@@ -44,6 +46,15 @@ SourcesApiClient.configure do |config|
   config.host   = "#{args[:sources_host]}:#{args[:sources_port]}"
 end
 
+ActiveSupport::Notifications.subscribe(/.*heartbeat\.consumer\.kafka.*$/) do |*options|
+  messaging_options = TopologicalInventory::AnsibleTower::Operations::Worker.default_messaging_opts
+  event = ActiveSupport::Notifications::Event.new(*options)
+  if messaging_options[:group_ref] == event.payload[:group_id] && messaging_options[:client_ref] == event.payload[:client_id]
+    heartbeat = TopologicalInventory::Providers::Common::Heartbeat.new("operations-kafka")
+    heartbeat.touch_heartbeat_file
```

Comment:
This is just a suggestion. For patterns like this, it may be cleaner to have a class method of the `Heartbeat` class memoize and cache the instance and perform the operation. For example:
```
TopologicalInventory::Providers::Common::Heartbeat.touch_heartbeat_file("operations-kafka")
```
Here, the `touch_heartbeat_file` class method would instantiate the instance and hash it based on name, if needed. Then perform the operation on the hashed value.

This comment applies more to https://github.com/RedHatInsights/topological_inventory-providers-common/pull/17, but the use case is more visible here.

Here again, this is only a suggestion, as this is a cosmetic issue.